packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderMenuIcon.tsx

```diff
@@ -39,6 +42,10 @@ export function ColumnHeaderMenuIcon(props: ColumnHeaderFilterIconProps) {
         title={apiRef!.current.getLocaleText('columnMenuLabel')}
         size="small"
         onClick={handleMenuIconClick}
+        aria-expanded={open ? 'true' : undefined}
```

Comment:
`open && 'true' `

Reply:
```suggestion
        aria-expanded={open ? true : undefined}
```

Reply:
I took that approach from the core and from the one we did before that.